keyboards/ramonimbao/herringbone/v1/readme.md

```diff
@@ -0,0 +1,15 @@
+# Herringbone
+
+![Herringbone](https://i.imgur.com/SQyS0j8.png)
+
+A through-hole 75% keyboard.
+
+* Keyboard Maintainer: [Ramon Imbao](https://github.com/ramonimbao)
+* Hardware Supported: ATmega32A
+* Hardware Availability: [Herringbone](https://github.com/ramonimbao/Herringbone)
+
+Make example for this keyboard (after setting up your build environment):
+
+    make ramonimbao/herringbone/v1:default
+
```

Comment:
Can you add "clear instructions on how to reset the board into bootloader mode" as per [PR Checklist](https://docs.qmk.fm/#/pr_checklist?id=keyboard-prs)?